### PR TITLE
mission: add missing result

### PR DESCRIPTION
**Pull request by julianoes**
Anything else that needs to be added for this?

### CODE REVIEW

File: protos/mission/mission.proto [hunk 1]
```diff
@@ -119,5 +119,6 @@ message MissionResult {
         FAILED_TO_OPEN_QGC_PLAN = 9;
         FAILED_TO_PARSE_QGC_PLAN = 10;
         UNSUPPORTED_MISSION_CMD = 11;
+        CANCELLED = 12;
```
Comment (JonasVautherin) [inline protos/mission/mission.proto]: Wait, does it mean "UPLOAD_CANCELLED"?

Comment (julianoes) [reply protos/mission/mission.proto]: Or download cancelled.

Comment (JonasVautherin) [reply protos/mission/mission.proto]: But not `MISSION_CANCELLED`, right? What about `TRANSFER_CANCELLED` then?

Comment (julianoes) [reply protos/mission/mission.proto]: Right that would be better but you'd have to change it here first:
https://github.com/Dronecode/DronecodeSDK/blob/develop/plugins/mission/include/plugins/mission/mission.h#L54

Comment (JonasVautherin) [reply protos/mission/mission.proto]: The gRPC server can make the translation until we update the C++ API. Given that the goal will be eventually to generate the C++ API from these proto files, right?

**Comment by JonasVautherin** [comment]
Should we do something about that? I still vote for `TRANSFER_CANCELLED`, and I volunteer to adapt the C++ part when the headers get generated :D

**Comment by julianoes** [comment]
Ok @JonasVautherin whatever you suggest.

**Comment by JonasVautherin** [comment]
@julianoes may I ask you to merge, as a sign for "approval"? It's too easy for me to edit your PR and then approve it :sweat_smile:.

**Pull request merged**